src/rulesets/reader.ts

```diff
@@ -79,12 +79,16 @@ const createRulesetProcessor = (
         dereferenceInline: false,
         uriCache,
         async parseResolveResult(opts) {
-          opts.result = parseContent(opts.result, opts.targetAuthority.pathname());
+          opts.result = parseContent(opts.result, opts.targetAuthority.valueOf());
```

Comment:
We could also use `normalize` from `@stoplight/path` to make sure paths are in tact.
```suggestion
          opts.result = parseContent(opts.result, normalize(opts.targetAuthority.valueOf()));
```

Reply:
and potentially in other spots we care about in `reader.ts`